test/e2e/chaos/timechaos/time_skew.go

```diff
@@ -215,3 +216,100 @@ func TestcaseTimeSkewPauseThenUnpause(
 	By("delete chaos CRD objects")
 	cli.Delete(ctx, timeChaos)
 }
+
+func TestcaseTimeSkewStartAtWaitingThenIntoRunning(
+	ns string,
+	cli client.Client,
+	c http.Client,
+	port uint16,
+) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	By("wait e2e helper ready")
+	err := util.WaitE2EHelperReady(c, port)
+	framework.ExpectNoError(err, "wait e2e helper ready error")
+
+	By("create cron")
+	minuteNow := time.Now().Minute()
+	crontab := fmt.Sprintf("%d * * * *", (minuteNow+3)%60)
```

Comment:
seems that this test will run at least 3 minutes, can we decrease the time cost?

Reply:
Actually `+3` means it would take 2-3 mins (say we do the plus at 59 seconds and it would only take 2 mins to start)
And since we need to wait at most 1 minutes to make sure chaos start and into wait phase, and another minute to make sure chaos not take effect, I think `+3` is the minimum time we need to wait. Or we need to cut out those "1 minutes" to lower duration say 30s, maybe that would import some unstability